src/game/engine/aircrafttype.cpp

```diff
@@ -193,6 +194,14 @@ AircraftTypeClass &AircraftTypeClass::As_Reference(AircraftType type)
     return g_AircraftTypes[type];
 }
 
+AircraftTypeClass *AircraftTypeClass::As_Pointer(AircraftType type)
+{
+    DEBUG_ASSERT(type != AIRCRAFT_NONE);
```

Comment:
These asserts don't mean anything here as the function already handles these cases and as such they are not exceptional circumstances that the function expects to never happen.